lib/client/BootstrappedDataService.js

```diff
@@ -58,6 +53,14 @@ BootstrappedDataService.prototype = {
 
 };
 
+BootstrappedDataService.computeKey = function(url, queryParams) {
+    if (queryParams) {
+        return url + JSON.stringify(queryParams);
```

Comment:
did you compare json stringify to creating a query string? i'm curious how they compare for performance and space?